### PR TITLE
chore(DropdownMenu): Make links black. Add external link icon onto ex…

### PR DESCRIPTION
**Pull request by jamiek-acl**
Before:
Screen Shot 2020-10-29 at 10 39 26 AM

After:
Screen Shot 2020-10-29 at 11 06 48 AM

### CODE REVIEW

File: packages/DropdownMenu/src/components/LinkItem/LinkItem.js [hunk 1]
```diff
@@ -33,15 +34,18 @@ const LinkItem = props => {
     ...moreProps,
   };
 
+  let externalLinkIcon = null;
   if (isExternal) {
     linkItemProps["aria-label"] = I18n.t("dropdownMenu.isExternal", { link: children });
     linkItemProps.target = "_blank";
     linkItemProps.rel = "noopener noreferrer";
+    externalLinkIcon = <NewTabIcon size="10px" />;
```
Comment (tristanjasper) [inline packages/DropdownMenu/src/components/LinkItem/LinkItem.js:42]: Perhaps we could leverage the `Button.Link` component 
```
if (isExternal) {
    linkItemProps.a11yText = I18n.t("dropdownMenu.isExternal", { link: children });
    linkItemProps.shouldOpenNewTab = true;
    linkItemProps.suffixIcon = null;
  }

  return (
    <Button.Link css={linkItemStyles} {...linkItemProps}>
      {children}
    </Button.Link>
  );
```

**Comment by tristanjasper** [comment]
Looks like this is digging into the issue I created recently - https://github.com/acl-services/paprika/issues/735
that myself and @krismckinnonacl were discussing.

File: packages/DropdownMenu/src/Item.styles.js [hunk 2]
```diff
@@ -5,12 +5,17 @@ import stylers from "@paprika/stylers";
 const itemStyles = css`
   ${stylers.fontSize()};
   ${stylers.lineHeight()}
+  color: ${tokens.color.black};
```
Comment (tristanjasper) [inline packages/DropdownMenu/src/Item.styles.js:8]: I know when I was consuming the dropdownMenu component in results and used the non external links I had application styles causing styling issues, so I was experimenting with adding 

```
border-radius: 0;
border-width: inherit;
min-height: inherit;
text-align: left;
``` 
 to fix some of those application styles from affecting this. But can revisit that later.

File: packages/DropdownMenu/src/components/LinkItem/LinkItem.js (hunk 1 above)
Comment (jamiek-acl) [reply packages/DropdownMenu/src/components/LinkItem/LinkItem.js:42]: Hmm I don't think that's necessary, I'm not experiencing any focus ring style issues as-is.

Comment (tristanjasper) [reply packages/DropdownMenu/src/components/LinkItem/LinkItem.js:42]: Ok np I will address the keyboard navigation focus bug (when first item is a link) in a follow up pr.

**Review by tristanjasper — APPROVED** [review]

**Pull request merged**